releaseNotes/MicrosoftAzureAppConfigurationAspNetCore.md

```diff
@@ -1,6 +1,13 @@
 ## Microsoft.Azure.AppConfiguration.AspNetCore
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Azure.AppConfiguration.AspNetCore)
 
+### 4.0.0-preview - July 23, 2020
+* Added multi-targeting support for .NET Core 3.1 besides .NET Standard 2.0. [#173](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/173)
+* Extracted out the code to retrieve required dependencies for configuration refresh from `IApplicationBuilder.UseAzureAppConfiguration()` to `IServiceCollection.AddAzureAppConfiguration()` in the package `Microsoft.Extensions.Configuration.AzureAppConfiguration`, so it could be re-used for non ASP.NET Core applications.
```

Comment:
What you said is not wrong, but that's the implementation detail. How about this?

**Breaking Change:** To leverage the new feature of dependency injection support for obtaining `IConfigurationRefresher` instances introduced in the 4.0.0-preview version of the `Microsoft.Extensions.Configuration.AzureAppConfiguration` package, the following changes are made.